.github/workflows/build_coverage.yml

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-name: "Build / Coverage (master, Scala 2.12, Hadoop 3, JDK 8)"
+name: "Build / Coverage (master, Scala 2.12, Hadoop 17, JDK 8)"
```

Comment:
NO.. dont change hadoop from 3 to 17 but JDK 8 to 17

Reply:
good catch ~ thanks